http/client/src/main/java/org/wildfly/security/http/client/mechanism/bearer/ElytronHttpClientBearerAuthMechanism.java

```diff
@@ -0,0 +1,26 @@
+package org.wildfly.security.http.client.mechanism.bearer;
+
+import org.wildfly.security.http.client.utils.ElytronHttpClientCredentialUtils;
+
+import java.net.URI;
+import java.net.http.HttpRequest;
+
+public class ElytronHttpClientBearerAuthMechanism {
+
+    private static final String AUTHORIZATION = "Authorization";
```

Comment:
@keshav-725 We use the AUTHORIZATION constant in all mechanisms. You should you create some constant that will be shared between all